gnomad/utils/release.py

```diff
@@ -0,0 +1,94 @@
+from typing import Dict, List, Optional, Union
```

Comment:
I don't think Union is used anymore

```suggestion
from typing import Dict, List, Optional
```